test/utils/specUtils.js

```diff
@@ -8,3 +8,51 @@ function eventFire(el, etype){
     el.dispatchEvent(evObj);
   }
 }
+
+angular.module('patternfly.testutils',['patternfly.utils'])
```

Comment:
What about putting this into a different file within the utils directory?  That way it's contained since this file seems to be for common test utils.